labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -65,7 +98,7 @@ object CollectionExercise03 {
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
   def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
+    seq == seq.sorted
```

Comment:
I did a pattern matching approach to this question. I'm not sure if it would work for the example in the header (1,2,2). If I'm not mistaken, it seems like it'd just check if (1,2,2) was == to (1,2,2) (already sorted). This might work if all the values were unique. Because the tests aren't too robust, it seems there is a good amount of leeway for passing all the tests.